osdk/src/cli.rs

```diff
@@ -109,6 +115,48 @@ pub struct RunArgs {
     pub cargo_args: CargoArgs,
     #[command(flatten)]
     pub osdk_args: OsdkArgs,
+    #[command(flatten)]
+    pub gdb_server_args: GdbServerArgs,
+}
+
+#[derive(Debug, Args, Clone, Default)]
+pub struct GdbServerArgs {
+    /// Whether to enable QEMU GDB server for debugging
+    #[arg(
+        long,
+        short = 'G',
+        help = "Enable QEMU GDB server for debugging",
+        default_value_t
+    )]
+    pub gdb_server: bool,
+    #[arg(
+        long = "vsc",
+        help = "Generate a '.vscode/launch.json' for debugging with Visual Studio Code \
+                (only works when '--gdb-server' is enabled)",
+        default_value_t
+    )]
+    pub vsc_launch_file: bool,
+    #[arg(
+        long = "gdb-stub",
+        help = "Path of GDB server stub",
```

Comment:
"The network address on which the GDB server listens. The address can be either a path for the UNIX domain socket or an IP address."